docs/admin-guide.md

```diff
@@ -1,35 +1,39 @@
 # Administrator Guide
 
-- [Src Folder](#src-folder)
-- [adfconfig](#adfconfig)
-  - [Roles](#roles)
-  - [Regions](#regions)
-  - [Config](#config)
-- [Accounts](#Accounts)
-  - [Master](#master-account)
-  - [Deployment](#deployment-account)
-    - [Default Deployment Account Region](#default-deployment-account-region)
-  - [Bootstrapping](#bootstrapping-accounts)
-    - [Bootstrapping Overview](#bootstrapping-overview)
-    - [Bootstrapping Inheritance](#bootstrapping-inheritance)
-    - [Regional Bootstrapping](#regional-bootstrapping)
-    - [Global Bootstrapping](#global-bootstrapping)
-    - [Bootstrapping Regions](#bootstrapping-regions)
-    - [Bootstrapping Recommendations](#bootstrapping-recommendations)
-  - [Pipelines](#pipelines)
-    - [Pipeline Parameters](#pipeline-parameters)
-    - [Using Github](#using-github)
-    - [Chaining Pipelines](#chaining-pipelines)
-- [Service Control Policies](#service-control-policies)
-- [Tagging Policies](#tagging-policies)
-- [Pipelines](#pipelines)
-  - [Pipeline Parameters](#pipeline-parameters)
-  - [Chaining Pipelines](#chaining-pipelines)
-- [Integrating Slack](#integrating-slack)
-- [Check Current Version](#check-current-version)
-- [Updating Between Versions](#updating-between-versions)
-- [Removing ADF](#removing-adf)
-- [Troubleshooting](#troubleshooting)
+- [Administrator Guide](#administrator-guide)
```

Comment:
Can you rerun this table to include the latest sections added by the PRs that were open alongside this one?

Reply:
👍 done